content/hardware/01.mkr/01.boards/mkr-wan-1300/tutorials/lora-button-press/lora-button-press.md

```diff
@@ -9,7 +9,7 @@ tags:
   - LED
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®
```

Comment:
Thats the library name, so we should sync with the library's name
```suggestion
  - name: LoRa
```

Reply:
changed